vuepress/docs/next/tutorials/devops/installation/elastic-kubernetes-service/eks-install.md

```diff
@@ -32,59 +30,70 @@ These steps will use the AWS console to create the cluster. If you’re already
+   - Go to `IAM → Roles → my-eks-role`
+   - Under Permissions click `Attach policies`
    - Add a policy of `AmazonEKSWorkerNodePolicy`
    - Add a policy of `AmazonEKS_CNI_Policy`
    - Add a policy of `AmazonEC2ContainerRegistryReadOnly`
    - Add a policy of `ElasticLoadBalancingFullAccess`
-   - Make sure `"Service": "ec2.amazonaws.com"` is in the Trust Relationship to use the same role for Node Group
+   - Under Trust Relationships click `Edit trust relationship`. Add `ec2.amazonaws.com` so the cluster can manage the EC2 resources. 
+```
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+```
 4. Go to `Services` and select `Elastic Kubernetes Service`
 5. Create an EKS Cluster
    - Add a cluster name (e.g. `cluster-1`) and click `Create EKS cluster`
-   - Select `1.18` for the Kubernetes version
-   - For `Cluster Service Role`, select the role you created above, e.g. `my-eks-role`. If you choose a different role it must have ELB permissions so the cluster can create a load balancer in `Networking` (Step 2).
+   - Select an [Entando-compatible Kubernetes version](https://www.entando.com/page/en/compatibility-guide), e.g. `1.20`
+   - For `Cluster Service Role`, select the role you created above, e.g. `my-eks-role`. If you choose a different role it must have ELB permissions so the cluster can create a load balancer in `Networking`.
    - Click `Next`
-   - Use the defaults for `Networking` (Step 2) and click `Next`
-   - Use the defaults for `Configure Logging` (Step 3) and click `Next`.
+   - Use the defaults for `Networking` and click `Next`
+   - Use the defaults for `Configure Logging` and click `Next`.
    - Review your settings and then click `Create`. Cluster provisioning usually takes between 10 and 15 minutes.
    - See <https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html> for more information on cluster creation.
-
-6. Add a node group to the cluster
-   - Go to `Services → Elastic Kubernetes Service → Clusters` → Click on your cluster name.
-   - Go to the `Compute` tab
+6. Add a Node Group to the cluster
+   - Go to `Services → Elastic Kubernetes Service → Clusters` and click on your cluster name.
+   - Go to `Configuration` and the `Compute` tab
    - Click `Add Node Group`
    - `Name`: give your group a name, e.g. `node-1`
    - `Node IAM Role`: Select the cluster role you created above. If the role doesn't appear, verify that you added the extra policies to the role.
-   - `Subnets` - VPC subnets should already be setup and selected.
-   - Select `Allow remote access to nodes`.  Follow the links to create a new SSH key pair if you don't already have one.
    - Click `Next`
+   - Review the compute and scaling configuration. Typically the AWS defaults should work.
    - AMI type: `Amazon Linux 2`
```

Comment:
Unsure but should compute or scaling be capitalized?

Reply:
Done